src/onyx/kafka/information_model.cljc

```diff
@@ -79,10 +79,20 @@
               :default 2000
               :optional? true}
 
+             :kafka/deserializer
```

Comment:
`:kafka/deserializer` -> `:kafka/value-deserializer` for consistency with kafka.